pom.xml

```diff
@@ -45,5 +45,6 @@
         <module>dynamodb-client</module>
         <module>using-oauth2-rbac</module>
         <module>artemis-jms</module>
+        <module>using-stm</module>
```

Comment:
I've found the name not necessarily self-descriptive. Could we come up with something a bit more expressive?

Reply:
Something like `using-software-transactional-memory`.

Reply:
+1